presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -283,6 +283,17 @@ Spilling Properties
     this property defines the threshold at which to trigger spilling for a task.  This property
     is ignored for any other spilling strategy.
 
+``experimental.max-revocable-memory-per-node``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    * **Type:** ``data size``
+    * **Default value:** ``16GB``
+
+    This is the amount of memory that is being used by a query that can be spilled to disk.
+    A query can request memory that does not count toward the limits, but this memory can be
+    revoked by the memory manager at any time.
+
+    This config property's value is how much revocable memory per node any one query is allowed to use.
```

Comment:
s/This config property's value is how much/This property defines the amount of/

Reply:
Done